observability/pillars/metrics.md

```diff
@@ -42,6 +42,7 @@ System critical measurements that relate to the application/machine health, whic
 - Request rate.
 - Queue length.
 - Unexpected exception count.
+- Dependent service metrics like response time for Redis cache, Sql server or Service bus.
```

Comment:
Semi-nit: A suggestion I would make is instead of using the words "dependent service", use "external service", which is more generic. 

For me, a dependent service is one that would break my system if it goes down. An external service could be a strong dependency or a softer one (meaning that your system can keep running even if it goes down), and I believe both should be tracked.

Reply:
Thanks @ivaramme for review. Few points here:
1) For micro service environment where one service (S1) calls another service(S2), in that case S2 can be considered as an internal service(since development team owns code for S2) but S1 is still dependent on S2. Here we can also name S2 as internal dependent service or just a `dependent service`.
2) Services like Redis Cache or Service Bus can be considered as external service, which as you mentioned can be a strong or soft dependency. 
Weather its external or internal, or have a strong or soft dependency, its advisable to monitor all of them. 
I was trying to refer to all set of dependent services.  But please suggest  if it still makes sense to rename it to `external service`

Reply:
Thanks, @anandchugh - Yes, agreed about monitoring both. That's why I suggested using `external`, to cover the two cases.